src/tests/androidTestRunner.ts

```diff
@@ -0,0 +1,38 @@
+import * as path from "path";
+import AppCenterUITestRunner from "./appCenterUITestRunner";
+
+export class AndroidTestRunner extends AppCenterUITestRunner {
+
+    protected getBundleName(): string {
+        return `index.android.bundle`;
+    }
+
+    protected getBundleOutputDir(): string {
+        return `${this.getAssetsFolder()}/${this.getBundleName()}`;
+    }
+
+    protected getAssetsFolder(): string {
+        return "android/app/src/main/assets";
+    }
+
+    protected getAdditionalArgs(): string[] {
+        return ["--app-path", "app/build/outputs/apk/app-debug.apk"];
+    }
+
+    protected getAbsoluteBuildDirectoryPath(): string {
+        return path.join(this.nativeAppDirectory, 'app/build');
+    }
+
+    protected getRelativeBuildBinaryDirectoryPath(): string {
+        return "app/build/outputs/apk";
+    }
+
+    protected getTestFrameworkName(): string {
+        return "espresso";
```

Comment:
Other frameworks not supported? How hard is to add support for other frameworks?

Reply:
As far as I see, others are pretty difficult